apps/src/gamelab/GameLabSprite.js

```diff
@@ -213,7 +204,7 @@ module.exports.createSprite = function (x, y, width, height) {
    * @method
    */
   s.isTouching = function (target) {
-    return this.createGroupState('collide', target, undefined, false);
+    return this.createGroupState('overlap', target);
```

Comment:
Should this code be functionally equivalent? I remember there used to be some slight differences in boundary conditions (adjacent vs. overlapping), but I don't remember the details. I do recall that we want the student to be able to write something like this:
```
if (sprite.isTouching(sprite2)) {
  sprite.collide(sprite2);
}
```